.changeset/angry-dingos-run.md

```diff
@@ -0,0 +1,5 @@
+---
+'sv': patch
+---
+
+chore: upgrade SvelteKit and add prepare script to generate tsconfig for local installs
```

Comment:
This PR is no longer upgrading SK. And it's also not just for tsconfig - it's anything else that we might want in `.svelte-kit`. I don't know whether that's important to be more precise about.